frontend/src/api/helm/getRelease.ts

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+import {HELM_API_URL} from "../../consts/consts.ts";
+import {HelmRelease} from "../../types";
+
+export async function fetchRelease(releaseName: string, namespace: string): Promise<HelmRelease> {
```

Comment:
This function is not used anywhere

Reply:
Yes, I haven't used it yet. I don't want to redo details later.